### PR TITLE
BC-857 cleanup Dockerfile and surroundings

### PR DESCRIPTION
**Pull request by Loki-Afro**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -5,3 +5,6 @@ deploy
 node_modules
 .travis.yml*
 Dockerfile.*
+
+**/*.unit.ts
+**/*.unit.js
```
Comment (CeEv) [inline .dockerignore:10]: The linter want a line break.

**Review by CeEv — CHANGES_REQUESTED** [review]

**Review by CeEv — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+USER node
+RUN npm ci
+COPY --chown=node:node locale /app/locale
+COPY --chown=node:node src /app/src
+
+ENV NODE_ENV=production
+RUN ["npm", "run", "build:nuxt"]
+COPY --from=git /app/nuxtversion /app/src/static/nuxtversion
+COPY run.sh /app
+CMD [ "./run.sh"]
```
[the remark this answers is not in the document]
Comment (Loki-Afro) [reply Dockerfile:29]: linter should not care about Dockerfile

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
+FROM docker.io/node:16 as git
+
+RUN mkdir /app && chown -R node:node /app
+WORKDIR /app
+COPY package.json .
+COPY .git .
+RUN echo "{\"sha\": \"$(git rev-parse HEAD)\", \"version\": \"$(node -p "require('./package.json').version")\", \"commitDate\": \"$(git log -1 --format=%cd --date=format:'%Y-%m-%dT%H:%M:%SZ')\", \"birthdate\": \"$(date +%Y-%m-%dT%H:%M:%SZ)\"}" > /app/nuxtversion
+
+FROM docker.io/node:16
```
[the remark this answers is not in the document]
Comment (Loki-Afro) [reply Dockerfile:9]: so?

File: .dockerignore (hunk 1 above)
Comment (Loki-Afro) [reply .dockerignore:10]: linter should not care about .dockerignore

**Review by odalys-dataport — DISMISSED** [review]

**Review by CeEv — APPROVED** [review]

**Pull request merged**